### DISCUSSION TITLE
Welcome to pyaqsapi Discussions!

### DISCUSSION THREAD

**Discussion by mccroweyclinton-EPA** [USEPA/pyaqsapi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AMDeitsch** [comment]
Hi! I'm currently a PhD student studying "persistent pollutants" in the atmosphere. I'm actually working on a Cookbook for [Project Pythia](https://projectpythia.org/) that showcases the `pyaqsapi` module and how to use it, and was wondering if there was a proper logo for the AQS that could be called on. So far, the only one I've seen is [this](https://www.epa.gov/sites/default/files/styles/thumbnail/public/2015-09/aqs001_0.png?itok=9kI8EvbU). 

Thanks,

Adam

**Reply by mccroweyclinton-EPA** [reply]
That is the only logo that is available for AQS that I know of as of right now. Unfortunately, due to restrictions on the usage of that logo, the EPA cannot grant usage of that logo for any project that was produced outside of the EPA.

**Reply by AMDeitsch** [reply]
Thank you.
